hazelcast/src/main/java/com/hazelcast/multimap/impl/MultiMapContainer.java

```diff
@@ -55,6 +55,8 @@
     // these fields are volatile since they can be read by other threads than the partition-thread
     private volatile long lastAccessTime;
     private volatile long lastUpdateTime;
+    private volatile long hits;
+    private AtomicInteger size;
```

Comment:
I am aware that instead of an `AtomicInteger`, I could use a `AtomicIntegerFieldUpdater`. However I was not sure if that is needed given the expected number of `MultiMapContainer` instances in the JVM. Please let me know if I should make this change.